src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/UdtTest/UdtTest.cs

```diff
@@ -294,8 +294,6 @@ public void NullTest()
                         {
                             if(e.Message.Contains("Rerun the transaction"))
                                 rerun = true;
-                            else 
```

Comment:
Why are we swallowing the exception silently?

Reply:
Fixed, was due to .NET error.